benchmarks/serialization_libraries/compare.sh

```diff
@@ -0,0 +1,222 @@
+#!/usr/bin/env bash
+#
```

Comment:
I could put this anywhere

Is a utility that only needs to be run when changing the serialization or adding a new one, since the validation ensures the serialization doesn't change.